soccer/radio/NetworkRadio.cpp

```diff
@@ -0,0 +1,126 @@
+#include "NetworkRadio.hpp"
+#include <Utils.hpp>
+#include "Geometry2d/Util.hpp"
+#include "status.h"
+#include "PacketConvert.hpp"
+
+using namespace boost::asio;
+using ip::udp;
+
+NetworkRadio::NetworkRadio(int server_port)
+    : _socket(_context, udp::endpoint(udp::v4(), server_port)),
+      _send_buffers(6) {
```

Comment:
The 6 can be `Robots_Per_Team` https://github.com/RoboJackets/robocup-software/blob/7d4e49eb543f6d07b512f61065e2c415f48da4ba/common/Constants.hpp#L16